lib/hanami/commands/console.rb

```diff
@@ -22,7 +22,7 @@ def reload!
         'irb'  => 'IRB'
       }.freeze
 
-      DEFAULT_ENGINE = 'irb'.freeze
+      DEFAULT_ENGINE = ['irb'].freeze
```

Comment:
What do you think about leaving this constant as is and change the caller so that it wraps this constant to an array instead?